docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -726,6 +726,12 @@ There are performance implications but they are typically mild due to hardware a
 
 ---
 
+#### Additional Considerations
+
+Intel CET shadow stack requires Linux Kernel version 6.6 or higher and glibc version 2.39 or higher. Shadow stack support must, in addition, be enabled at run-time by setting the corresponding hardware capability tunable for glibc via the `GLIBC_TUNABLES` environmental variable [^glibc-tunables]: `export GLIBC_TUNABLES=glibc.cpu.hwcaps=SHSTK`.
```

Comment:
Maybe include IBT while at it? (https://www.gnu.org/software/libc/manual/html_node/Hardware-Capability-Tunables.html#index-glibc_002ecpu_002ex86_005fibt)